vitrina/datasets/models.py

```diff
@@ -430,6 +450,12 @@ def lt_description(self):
     def en_description(self):
         return self.safe_translation_getter("description", language_code="en")
 
+    def lt_subclass_title(self):
+        return self.subclass.safe_translation_getter("title", language_code="lt")
+
+    def en_subclass_title(self):
+        return self.subclass.safe_translation_getter("title", language_code="en")
```

Comment:
Let's add typehints ( I see that other methods in this class don't have, but let's still follow the good practices :) )